roles/openshift_node/tasks/gather_debug.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: Get service status
+  command: >
+    systemctl status {{ item }}
+  changed_when: false
+  register: systemctl_status
+  loop:
+  - cri-o
+  - kubelet
+
+- name: Print service status
+  debug:
+    var: systemctl_status
```

Comment:
I had to refresh my memory on [Ansible variables & loops](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#registering-variables-with-a-loop) to realize this will print the output for all iterations (rather than overwriting the register each time).